core/src/main/scala/org/typelevel/otel4s/otel4s.scala

```diff
@@ -151,6 +156,15 @@ object Counter {
     new Counter[F, A] {
       def add(value: A, attribute: Attribute[_]*) = F.unit
     }
+
+  implicit final class CounterSyntax[F[_], A](
```

Comment:
We'll eventually have to decide whether we want top-level implicits or an implicits object.  Current fashion seems to be split...